middlewares/validators/tasks.js

```diff
@@ -164,6 +164,19 @@ const getTasksValidator = async (req, res, next) => {
         })
       ),
     size: joi.number().integer().positive().min(1).max(100).optional(),
+    q: joi
+      .string()
+      .optional()
+      .custom((value, helpers) => {
+        if (value && value.includes(":")) {
+          const [key] = value.split(":");
+          const allowedKeywords = ["searchterm", "assignee", "status"];
```

Comment:
is assignee and status handled ?

Reply:
not now because the scope of the pr is focused on the task search term only but can create another issue for it

Reply:
done